magmartfront/src/APIs/editUser.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import {UserData} from '@/components/myTypes/UserTypes';
+function EditUser( id: string ) {
+    return Promise.resolve(axios.patch<UserData>(`http://localhost:3001/users/${id}/update`))
```

Comment:
Padronizar a chamada usando variável de ambiente.
```suggestion
    return Promise.resolve(axios.patch<UserData>(`${process.env.API_URL}/users/${id}/update`))
```